talk/owt/sdk/include/cpp/owt/base/globalconfiguration.h

```diff
@@ -65,6 +65,14 @@ class GlobalConfiguration {
      encoded_frame_ = enabled;
   }
   /** @endcond */
+  /**
+   @brief This function sets the video encoder to be a custom video encoder that
+   supports both raw and encoded video frames.
+   @param enabled Dual video encoder is enabled or not.
+   */
+  static void SetDualVideoEncoderEnabled(bool enabled) {
```

Comment:
this will be called in appliance repo [here](https://github.com/AmbientAI/appliance/blob/42851a312da9acf027ebbecad0cbc0850bf09cd4/streaming/webrtc_server/src/main.cc#L156)